docs/guides/upgrading-to-v1.md

```diff
@@ -0,0 +1,59 @@
+---
+title: Upgrading to v1
+parent: Guides
+nav_order: 1
+---
+
+# Upgrading to v1
+
+## Installation
+
+Install `@humanmade/webpack-helpers@latest` with [npm](http://npmjs.com), along with more updated versions of `webpack` and its attendant dependencies:
+
+```bash
+npm install --save-dev @humanmade/webpack-helpers@latest webpack@5 webpack-cli@4 webpack-dev-server@4
+```
+
+## Breaking Changes
+
+### Handling multi-config arrays in DevServer
+
+In a large project you may have a Webpack configuration that [exports multiple separate Webpack configuration objects](https://webpack.js.org/configuration/configuration-types/#exporting-multiple-configurations) in a multi-config array. With Webpack 4 and the pre-1.0 version of this helper library, you may have code which maps over these arrays and adds a `devServer` property to each one:
+
+```js
+```
+
+Previously, if you exported an array of configurations from your Webpack development config, each one could have an identical `devServer` property and Webpack DevServer would handle all configs with the same server. This let us
```

Comment:
Just noting that this seems unfinished. Good cliffhanger, though. 😅

Reply:
Gotta make sure they're excited to buy the next upgrade! 😓 May not have time to flesh this out for a bit, would love help (from you or anybody reading) in fleshing out the upgrade guide.